definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -6,7 +6,7 @@
     "Name": "Service application received",
     "Description": "Service application received",
     "DisplayOrder": 11,
-    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AosAwaiting",
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AosAwaiting;AosDrafted",
```

Comment:
Already applied to Release PR:
https://github.com/hmcts/div-ccd-definitions/pull/632/commits/74244ddbf7ee4b9723fe50fd3dbd68f9a0045afa